tests/test_ios_testflight_main_push_filter.py

```
@@ -647,46 +718,97 @@ def test_demo_history_skips_newer_internal_artifact() -> None:
         event_name="schedule",
-        schedule=IOS_SCHEDULES[1],
-        input_variant="",
+        schedule=IOS_SCHEDULES[0],
         prior_upload_pages=(
-            first_page,
-            (("demo-base-sha", "ios-testflight-build-metadata-demo"),),
+            older_uploads,
+            (("newest-sha", "ios-testflight-build-metadata"),),
         ),
-        prior_run_ids=tuple(range(1_000, 899, -1)),
+        prior_artifact_minutes_ago=tuple(range(200, 100, -1)) + (0,),
```

Ruff (0.16.0)

[warning] Consider `(*tuple(range(200, 100, -1)), 0)` instead of concatenation

Replace with `(*tuple(range(200, 100, -1)), 0)`

(RUF005)